### ISSUE TITLE
No longer receiving images

### ISSUE THREAD

**Issue by boris-petrov** [dequis/purple-facebook#443]
I no longer seem to be receiving images. Not even a notification. Nothing. I have to log in to Facebook to see that there is actually a new message. This started happening a few weeks ago maybe? Not exactly sure. How do I help you with figuring it out?

**Comment by SM0TVI** [comment]
Have a similar issue, not getting messages sent from other clients, especially pidging. The 'droid facebook app appears to work as it should though.
Have you tried installing the freshest version of the facebook plugin and its dependencies? Have done so myself and waiting to see what results it will have.

**Comment by boris-petrov** [comment]
Yes, I'm always on the latest version and this happens with it. Probably FB changed something, yes. Funny thing is that it doesn't happen every time but only sometimes. Any ideas?

**Comment by SM0TVI** [comment]
Not a clue. Buddy list syncs okay, I do see when people are typing (but not what they're sending) and I have a chat window open, but no messages. Did they change some constant or channel for normal messages?
*scratches head*

**Comment by SM0TVI** [comment]
**UPDATE #2** I can't even see what I'm typing myself in the chat window, much less those I am typing to.

**Comment by SM0TVI** [comment]
As of this night (2018-10-30 between 00:00 to 07:00 local time) I got a burst of messages from the previous hours and pidgin appears to work again on Facebook (just chatted with a friend). I don't think we're out of the woods yet, but this has the hallmarks of a "d'Oh" over at Facebook.
